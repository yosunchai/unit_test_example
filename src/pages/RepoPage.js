import React, { Component } from 'react'
import { View, Text } from 'react-native'

class RepoPage extends Component {
  render() {
    return (
      <View>
        <Text>
          Loading...
        </Text>
      </View>
    )
  }
}

export default RepoPage
