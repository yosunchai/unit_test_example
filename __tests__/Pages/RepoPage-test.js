import React from 'react'
import renderer from 'react-test-renderer'

import RepoPage from '../../src/pages/RepoPage'

describe('RepoPage', () => {
  it('RepoPage be able to render correctly', () => {
    const component = renderer.create(
      <RepoPage />
    )

    expect(component).toMatchSnapshot()
  })
})
