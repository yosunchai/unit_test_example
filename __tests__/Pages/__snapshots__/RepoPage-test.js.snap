// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RepoPage RepoPage be able to render correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Loading...
  </Text>
</View>
`;
